src/ResourceManager/Network/Commands.Network/RouteTable/Route/GetAzureRmRouteConfigCommand.cs

```diff
@@ -0,0 +1,65 @@
+// 
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// 
+
+// Warning: This code was generated by a tool.
+// 
+// Changes to this file may cause incorrect behavior and will be lost if the
+// code is regenerated.
+
+using Microsoft.Azure.Commands.Network.Models;
+using Microsoft.Azure.Management.Network.Models;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Network.Automation
+{
+    [Cmdlet("Get", "AzureRmRouteConfig", SupportsShouldProcess = true), OutputType(typeof(PSRoute))]
```

Comment:
@EvgenyAgafonchikov GET cmdlets shouldn't support `ShouldProcess`; I would update the generator to reflect this.